app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelMain.tsx

```diff
@@ -1,10 +1,51 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { cx } from '@emotion/css';
-import { connectionPanelMainContainer } from './ConnectionPanel.styles';
+import Tabs, { Tab, TabList, TabPanel } from '@atlaskit/tabs';
+import { connectionPanelMainContainer, connectionPanelMainTabs } from './ConnectionPanel.styles';
+import { ConnectedState } from '../StatusLabel/StatusLabel';
+import { NotConnectedState } from './NotConnectedState';
 
-const ConnectionPanelMain = (): JSX.Element => {
+export const Panel = ({
+	children,
+	testId
+}: {
+	children: ReactNode;
+	testId?: string;
+}) => (
+	<div className={cx(connectionPanelMainTabs)} data-testid={testId}>
+		{children}
+	</div>
+);
+
+type ConnectionPanelMainProps = {
+	connectedState: ConnectedState
+};
+
+const ConnectionPanelMain = ({ connectedState }: ConnectionPanelMainProps): JSX.Element => {
 	return (
-		<div className={cx(connectionPanelMainContainer)}>Connection panel main coming soon...</div>
+		<div className={cx(connectionPanelMainContainer)}>
+			{connectedState === ConnectedState.DUPLICATE
+				? <NotConnectedState connectedState={connectedState} />
+				: <Tabs
+					id="default"
+				>
+					<TabList>
+						{/* TODO - update (0) for connected state */}
+						<Tab>Recent events (0)</Tab>
+						<Tab>Set up guide</Tab>
+					</TabList>
+					<TabPanel>
+						{connectedState === ConnectedState.CONNECTED
```

Comment:
/supernit

Because there are multiple conditionals it would be easier if the indentation made that obvious

```
{
    connectedState === ConnectedState.CONNECTED
        ? <Panel>List of servers goes here</Panel>
        : <Panel><NotConnectedState connectedState={connectedState} /></Panel>
}
```

or

```
{
    connectedState === ConnectedState.CONNECTED
        ? <Panel>List of servers goes here</Panel>
        : (
            <Panel>
                <NotConnectedState connectedState={connectedState} />
            </Panel>
        )
}
```

Wrapping the component (second option) will likely be more extensible when the list of servers is inserted